utils/bump_version.py

```diff
@@ -42,19 +43,8 @@
 
 how = sys.argv[1]
 
-with open("pyproject.toml", encoding="utf-8") as f:
-    content = f.read()
-old_version = re.search(r'version = "(.*)"', content).group(1)  # pyright: ignore[reportOptionalMemberAccess]
-version = old_version.split(".")
-if how == "patch":
-    version = ".".join(version[:-1] + [str(int(version[-1]) + 1)])
-elif how == "minor":
-    version = ".".join(version[:-2] + [str(int(version[-2]) + 1), "0"])
-elif how == "major":
-    version = ".".join([str(int(version[0]) + 1), "0", "0"])
-content = content.replace(f'version = "{old_version}"', f'version = "{version}"')
-with open("pyproject.toml", "w", encoding="utf-8") as f:
-    f.write(content)
+subprocess.run(["uv", "version", "--bump", how]) 
+version = subprocess.run(["uv", "version", "--short"], capture_output=True).stdout.decode("utf-8")
```

Comment:
@dangotbanned this is a pretty solid diff if you ask me! Thanks for suggesting it